ArduCopter/sensors.cpp

```diff
@@ -166,6 +166,7 @@ void Copter::read_battery(void)
     // update motors with voltage and current
     if (battery.get_type() != AP_BattMonitor::BattMonitor_TYPE_NONE) {
         motors->set_voltage(battery.voltage());
+        AP_Notify::flags.battery_voltage = battery.voltage(); //for on-board oled display
```

Comment:
could we get rid of all these "//for on-board oled display" comments? It's very clear what's the purpose now and later on this comment will become outdated when we introduce e.g. text-to-speech.